ansible/roles/passenger/tasks/main.yml

```diff
@@ -48,7 +48,7 @@
       dest: /etc/apache2/ssl/intermediate.crt
     sudo: true
 
-  - name: Create Rails Apache config (SSL Enabled)
```

Comment:
we probably wouldn't deploy node under apache we'd probably use nginx.  And passenger is a rails (rack) specific app server so i dont know what you mean by "rails agnostic.."  You woudn't use passenger without rails